shared/src/commonTest/kotlin/vn/luongvo/kmm/survey/data/repository/AuthRepositoryTest.kt

```diff
@@ -50,4 +58,13 @@ class AuthRepositoryTest {
             awaitError() shouldBe throwable
         }
     }
+
+    @Test
+    fun `when saving token - it executes the local data source`() = runTest {
```

Comment:
@luongvo should we write the test case like usual?
```suggestion
    fun `When saving token, it executes the local data source`() = runTest {
```

Reply:
@manh-t KMM shared not rely on JVM so we can use backtick method name without some special chars https://slack-chats.kotlinlang.org/t/2454195/i-see-plenty-of-mentions-that-fun-backtick-in-function-names

Reply:
@luongvo good to know it 😁